sass/directives/_headers.scss

```diff
@@ -172,3 +172,28 @@
     }
   }
 }
+
+@mixin header--small {
+  max-height: 300px;
```

Comment:
will you please make this a variable next to the other header height variables: https://github.com/cb-talent-development/employer-style-base/blob/master/sass/variables/_sizing.scss#L72